io/disk/lvsetup.py

```diff
@@ -50,7 +50,7 @@ def setUp(self):
         """
         pkg = ""
         smm = SoftwareManager()
-        self.disk = self.params.get('disks', default=None)
+        self.disk = self.params.get('lv_disks', default=None)
```

Comment:
having common name like "disk" will help for creating input file right?
any reason for changing to this name?

Reply:
When we give disks for lv, and then use the lv as input for other tests, we need different name.
That is the reason.

Reply:
that can be handled in script right?

Reply:
@narasimhan-v which script ?

Reply:
I mean using the different names for lvs inside the script if needed and we will stick with yaml naming concerns.

Reply:
@Naresh-ibm I think I can explain it better in person, let me do that when we meet.